test/e2e/scripts/local-deploy.sh

```
@@ -0,0 +1,1511 @@
+  _check() {
+    TOTAL=$((TOTAL + 1))
+    if eval "$2" &>/dev/null; then
+      ok "$1"; PASS=$((PASS + 1))
+    else
+      fail "$1"
+    fi
+  }
+
+  # Pods
+  echo -e "${BOLD}Pods${NC}"
+  _check "All pods running" '[[ $(kubectl get pods -A --no-headers | grep -v Running | grep -v Completed | wc -l) -eq 0 ]]'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)